tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -211,7 +236,7 @@ describe(__filename, () => {
     expect(root.find('.AddonMoreInfo-license')).toHaveLength(0);
   });
 
-  it('does not prefix a license link with the add-ons URL', () => {
+  it('does not prefix a non-custom license link with the add-ons URL', () => {
```

Comment:
The reason it says "add-ons URL" and not "add-on URL" is because it is referencing the URL for the add-ons (i.e., AMO) site, not a particular add-on URL. I will update the description to make it less ambiguous.